cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/RunExecutableTask.java

```diff
@@ -158,49 +148,17 @@ private void runGeneratedExecutable(BLangPackage executableModule, BuildContext
         }
     }
 
-    /**
-     * Run an executable that is generated from 'run bal' command.
-     *
-     * @param executableModule The module to run.
-     */
-    private void runGeneratedExecutableWithSameClassLoader(BLangPackage executableModule, BuildContext buildContext) {
-
-        ExecutableJar executableJar = buildContext.moduleDependencyPathMap.get(executableModule.packageID);
-        String initClassName = BFileUtil.getQualifiedClassName(executableModule.packageID.orgName.value,
-                executableModule.packageID.name.value,
-                MODULE_INIT_CLASS_NAME);
-        try {
-            URL[] urls = new URL[executableJar.moduleLibs.size() + 1];
-            urls[0] = buildContext.getJarPathFromTargetCache(executableModule.packageID).toUri().toURL();
-            int i = 1;
-            for (Path platformLib : executableJar.moduleLibs) {
-                urls[i++] = platformLib.toUri().toURL();
-            }
-            URLClassLoader classLoader = new URLClassLoader(urls);
-            Class<?> initClazz = classLoader.loadClass(initClassName);
-            Method mainMethod = initClazz.getDeclaredMethod(JAVA_MAIN, String[].class);
-            mainMethod.invoke(null, (Object) this.args);
-            if (!initClazz.getField("serviceEPAvailable").getBoolean(initClazz)) {
-                Runtime.getRuntime().exit(0);
-            }
-        } catch (MalformedURLException e) {
-            throw createLauncherException("loading jar file failed with given source path " + this.executableJarPath);
-        } catch (ClassNotFoundException e) {
-            throw createLauncherException("module init class with name " + initClassName + " cannot be found ");
-        } catch (NoSuchMethodException e) {
-            throw createLauncherException("main method cannot be found for init class " + initClassName);
-        } catch (IllegalAccessException | IllegalArgumentException e) {
-            throw createLauncherException("invoking main method failed due to " + e.getMessage());
-        } catch (InvocationTargetException | NoSuchFieldException e) {
-            throw createLauncherException("invoking main method failed due to ", e.getCause());
-        }
-    }
-
     private String getAllClassPaths(BLangPackage executableModule, BuildContext buildContext) {
```

Comment:
Shall we make this method to get the parameters instead of calculating inside method which would be more clear.

ex. getAllClassPaths(BLangPackage executableModule, boolean isSingleFileContext, String ballerinaHome)